src/afl-fuzz-python.c

```diff
@@ -220,14 +220,60 @@ static py_mutator_t *init_py_module(afl_state_t *afl, u8 *module_name) {
   if (py_module != NULL) {
 
     u8 py_notrim = 0;
+  #if PY_MINOR_VERSION > 12 || PY_MAJOR_VERSION > 3
```

Comment:
Why or the two? Shouldn't it be `&&`?
Did you try it on python < 3.13?

Reply:
It works without this patch with 3.12 and I don’t want to test where this would break with the new code. This works.
We don’t need && because it won’t compile with python 2, but at some point there will be python 4